### PR TITLE
Feature/yellow labs grafana

### PR DESCRIPTION
**Pull request by clemstoclems**
This is the first stable version for docker-compose stack

### CODE REVIEW

**Review by PurpleBabar — COMMENTED** [review]
je vais valider parceque t'es beau, mais voila les changes que j'aurais fait:
- stockage physique plutot que dans les volumes docker (pour eviter de perdre de la data sensible)
- je bosserai pas avec une version latest, on sait jamais que y'ai une major qui sorte ca peux te gener apres :p 
- j'aurais aussiptet remappé les ports histoire que des petits malins n'essayent pas de jouer avec les ports "classiques" des apps (coucou les filous)

Citou :)

**Comment by clemstoclems** [comment]
@PurpleBabar thanks for the review.

You are totally right for the ports mapping. But as it is a public repo, what do you think about using env vars for ports mapping, injected by the orcherstator ? 

Sure about the latest :) i'll fix it quickly.

About the volumes, for now i need them to dont be physical as we are still in development and testing. But be sure this will be fixed and release in another feature :)

**Comment by PurpleBabar** [comment]
Well my dear, injecting files from a stockage server with the orchestrator would be the optimum solution ;)

Cheers

**Comment by PurpleBabar** [comment]
The .env is commited oO , maybe adding it to the .gitignore would prevent it ;)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM httpd:2.4
+LABEL maintainer="Clément Guinet <cguinet@omnilog.fr>"
+
+
+COPY . /data/www/
+
+RUN apt-get update
```
Comment (PurpleBabar) [inline Dockerfile:7]: You can do it in one RUN ;)

**Pull request merged**